modules/statics/src/coins.ts

```diff
@@ -1237,56 +1237,13 @@ export const coins = CoinMap.fromCoins([
   avaxErc20('avaxc:link', 'Chainlink', 18, '0x5947bb275c521040051d82396192181b413227a3', UnderlyingAsset['avaxc:link']),
   avaxErc20('avaxc:usdt', 'Tether USD', 6, '0x9702230a8ea53601f5cd2dc00fdbc13d4df4a8c7', UnderlyingAsset['avaxc:usdt']),
   avaxErc20('avaxc:usdc', 'USD Coin', 6, '0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e', UnderlyingAsset['avaxc:usdc']),
-  tavaxErc20('tavaxc:qi', 'Test BenQi', 18, '0x8729438eb15e2c8b576fcc6aecda6a148776c0f5', UnderlyingAsset['avaxc:qi']),
```

Comment:
Hey Michael, these tokens are not out in production. I'm the lead for Avalanche, tokens project is still in development so it will not cause any interruptions.

Reply:
Sweet, so the removal then will be from the start, definitions not offered as part of our open source offering. LGTM.